docs/docsite/rst/getting_started_ee/index.rst

```diff
@@ -0,0 +1,36 @@
+.. _getting_started_ee_index:
+
+###########################################
+Getting started with Execution Environments
+###########################################
+
```

Comment:
Please adapt all your headers to match the Ansible style guide - https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#header-notation

Specifically, this should be *** over and under and all other rst files should start with === underline.